From fa3610449b5f27a54241339a876caee898b815e1 Mon Sep 17 00:00:00 2001
From: Tom White <tom@cloudera.com>
Date: Fri, 2 Dec 2011 16:48:03 -0800
Subject: [PATCH 008/795] MR1: Fix to allow tar build.
 (cherry picked from commit 14b4948fdd4efedfd5e21aace4a48aad52352a69)

(cherry picked from commit 6e8c3c2c4a0e35d3162e98b0b0e2b4723dd9996f)
---
 build.xml |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/build.xml b/build.xml
index e559dc1..8b3f44e 100644
--- a/build.xml
+++ b/build.xml
@@ -1081,7 +1081,6 @@
          <param name="-apidir" value="${jdiff.xml.dir}"/>
          <param name="-apiname" value="hadoop ${version}"/>
        </doclet>
-       <packageset dir="src/core"/>
        <packageset dir="src/mapred"/>
        <packageset dir="src/tools"/>
        <classpath >
-- 
1.7.0.4

